utilities/helperFunctions.js

```diff
@@ -75,6 +77,27 @@ const updateModMessage = (status, channel_id, text, user_id, ts, moderator) => {
     });
     return;
   }
+
+  const randomEmoji = (sentiment) => {
+    let emojis = [];
+    switch(sentiment){
```

Comment:
Please run this through ESLint. There should be a space between the `)` and the `{` if it's running correctly.

Reply:
Hmm I thought I had it on but I will double check